homeassistant/components/signal_messenger/notify.py

```diff
@@ -16,13 +16,15 @@
 CONF_SENDER_NR = "number"
 CONF_RECP_NR = "recipients"
 CONF_SIGNAL_CLI_REST_API = "url"
+CONF_IS_GROUP = "is_group"
 ATTR_FILENAME = "attachment"
 ATTR_FILENAMES = "attachments"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_SENDER_NR): cv.string,
         vol.Required(CONF_SIGNAL_CLI_REST_API): cv.string,
+        vol.Optional(CONF_IS_GROUP, default=False): cv.boolean,
```

Comment:
We no longer allow new integrations to use a platform config section or old integrations to do changes to the platform config section. See https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md.

Please refactor this integration to have the config schema in the base integration module and the config yaml section under the top level integration domain key.